esx_service/vmdk_ops.py

```diff
@@ -787,6 +790,7 @@ def get_datastore_name(datastore_url):
                       datastore_name)
```

Comment:
Should be logging.debug("get_datastore_name: path to /vmfs/volumes/%s does not exist", datastore_name)